projects/plugins/jetpack/_inc/client/components/section-nav/style.scss

```diff
@@ -419,14 +423,3 @@
 		padding: 3px 16px 5px;
 	}
 }
-
-// -------- Search --------
```

Comment:
Similar to the above, while this can be combined above, we lose the sectioning. 🤷

Reply:
d4eae44d0a